frontend/packages/console-shared/src/components/drawer/__tests__/Drawer.spec.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import Drawer from '../Drawer';
+import { DraggableCore } from 'react-draggable';
+import { CSSTransition } from 'react-transition-group';
+
+describe('DrawerComponent', () => {
+  it('should exist', () => {
+    const wrapper = shallow(<Drawer />);
+    expect(wrapper.find('div').exists()).toBe(true);
+  });
+
+  it('should have default values', () => {
+    const wrapper = shallow(
+      <Drawer>
+        <p>This is content</p>
+      </Drawer>,
+    );
+    expect(wrapper.find('div').exists()).toBe(true);
+    expect(wrapper.find(DraggableCore).exists()).toBe(false);
+    const style = wrapper
+      .find(CSSTransition)
+      .childAt(0)
+      .prop('style');
+    expect(style.height).toEqual(300);
```

Comment:
Perhaps you want to do this:
```suggestion
    const style = wrapper
      .find('.ocs-drawer')
      .prop('style');
    expect(style.height).toEqual(300);
```

Which keeps you from digging into a 3rd party component & goes straight to testing that a default component renders the default height.

Reply:
Same goes for any reference to this in the future tests.